src/app/backend-api/secom/model/searchObjectResult.ts

```diff
@@ -20,7 +20,7 @@ export interface SearchObjectResult {
     organizationId: string;
     endpointUri: string;
     endpointType: string;
-    keywords?: Array<string>;
+    keywords: string;
```

Comment:
Shouldn't this still be marked as optional, as the multiplicity is defined as 0..1 in SECOM?
```suggestion
    keywords?: string;
```